engine-rest/engine-rest-openapi/src/main/templates/paths/user/options.ftl

```diff
@@ -0,0 +1,27 @@
+{
+
+  <@lib.endpointInfo
+      id = "optionsUser"
+      tag = "User"
+      desc = "The /user resource supports two custom OPTIONS requests, one for the resource as such and one for individual user instances. The OPTIONS request allows checking for the set of available operations that the currently authenticated user can perform on the /user resource. If the user can perform an operation or not may depend on various things, including the user’s authorizations to interact with this resource and the internal configuration of the process engine." />
+
+  "responses" : {
+
+    <@lib.response
+        mediaType = "application/json"
+        code = "200"
+        dto = "AtomLink"
+        last = true
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "GET `/user/aUserId`",
```

Comment:
Please review the use of Atom link and the response in the example. 
I did my best here, but I'm not 100% confident.